cosmos/providers/dbt/core/operators.py

```diff
@@ -146,25 +148,33 @@ def get_env(self, context):
         return env
 
     def get_dbt_path(self):
-        bash_path = shutil.which("bash") or "bash"
+        which_dbt = shutil.which("dbt-ol") or shutil.which("dbt") or "dbt"
 
-        if self.python_venv:
-            dbt_path = [bash_path, "-c", f"source {self.python_venv} && dbt {self.base_cmd} "]
+        if self.dbt_executable_path:
+            dbt_path = self.dbt_executable_path
         else:
-            dbt_path = [bash_path, "-c", f"dbt {self.base_cmd} "]
+            dbt_path = which_dbt
```

Comment:
do we still need this `which_dbt` var or can we just use the `dbt_executable_path`?

Reply:
I'd leave this personally, but an open to having my mind changed. This will pick up any `dbt-ol` or `dbt` installations by default (outside of a venv). Leaving it this way gives our customers more flexibility (i.e. you want the venv, you specify the path -- otherwise, just add dbt-core to your requirements.txt)